config/default.js

```diff
@@ -6,7 +6,7 @@ const DAY_IN_MILLISECONDS = 1000 * 60 * 60 * 24;
 
 export default {
   api: {
-    root: 'https://candy.freefeed.net',
+    root: 'https://freefeed.net',
```

Comment:
You should not change the default settings, they purposely specify a stage instance (candy).

For testing purposes, you can override them in the config.json file at the root of the repository. It will not be included in the commit by .gitignore.